pkg/fission-cli/cmd/spec/apply.go

```diff
@@ -185,6 +188,27 @@ func (opts *ApplySubCommand) run(input cli.Input) error {
 	return nil
 }
 
+func warnIfDirtyWorkTree(path string) {
+	repo, err := git.PlainOpen(path)
+	if err != nil {
+		return
```

Comment:
The function should not ignore error silently. Can you please return an error from all places, let the caller decide how to handle the error.